x-pack/plugins/transform/server/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PluginInitializerContext } from 'src/core/server';
+
+import { TransformServerPlugin } from './plugin';
+import { configSchema } from './config';
+
+export const plugin = (ctx: PluginInitializerContext) => new TransformServerPlugin(ctx);
+
+export const config = {
+  schema: configSchema,
+};
```

Comment:
I copied this from the `index_management` plugin. I'm not sure this is applicable to the transforms plugin?